index.html

```diff
@@ -387,6 +388,75 @@ <h3>LinkedDataSignature2016</h3>
           </pre>
       </section>
 
+      <section id="Ed25519Signature2018">
+
+        <h3>Ed25519Signature2018</h3>
+          <p>
+A Linked Data signature is used for digital signatures on RDF Datasets.
+The default canonicalization mechanism is specified in the RDF Dataset
+Normalization specification, which effectively deterministically names all
+unnamed nodes. The default signature mechanism
+uses a SHA-256 digest and JWS to perform the digital signature.
+          </p>
+          <dl>
+            <dt>Status</dt>
+            <dd property="vs:term_status">stable</dd>
+            <dt>Parent Class</dt>
+            <dd>JwsLinkedDataSignature</dd>
```

Comment:
```suggestion
            <dd>Signature</dd>
```

There is no such thing as a JwsLinkedDataSignature defined, IIRC.